apps/service-portal/src/components/Header/Header.tsx

```diff
@@ -114,6 +114,7 @@ export const Header = ({ position }: Props) => {
                               type="span"
                               variant="utility"
                               unfocusable
+                              aria-label={formatMessage(m.openDocuments)}
```

Comment:
Do we need this since we have this already:

```
<span className={helperStyles.srOnly}>
                              {formatMessage(m.openDocuments)}
                            </span>
```

Reply:
Hmm prob not :D I'll remove it